scripts/in_nic/vocab/States.csv

```diff
@@ -0,0 +1,37 @@
+,state_id,state_name
```

Comment:
we should prefer filenames to be all lower cased.  states.csv instead of States.csv